src/config.js

```diff
@@ -33,8 +33,16 @@ module.exports = {
     hostname: process.env.ANALYTICS_HOSTNAME || "",
   },
 
+  postgres: {
+    host : process.env.POSTGRES_HOST,
+    user : process.env.POSTGRES_USER,
+    password : process.env.POSTGRES_PASSWORD,
+    database : process.env.POSTGRES_DATABASE || "analytics-reporter",
+  },
+
   static: {
-    path: '../analytics.usa.gov/'
+    path: '../analytics.usa.gov/',
   },
 
+  timezone: process.env.ANALYTICS_TIMEZONE || "US/Eastern",
```

Comment:
The generated data will not change regardless of how you set the timezone. That will remain as it was before, which is the timezone that Google Analytics is configured to use.

The reason to add a timezone config, is that the generated data we are writing has no associated timezone info. As a result, a date that should be `2017-01-01T5:00-5:00` gets written to the database as `2017-01-01T10:00Z`. A configuration for the timezone allows us to correct for that.